src/Build/Logging/ParallelLogger/ParallelConsoleLogger.cs

```diff
@@ -914,6 +962,22 @@ public override void TaskFinishedHandler(object sender, TaskFinishedEventArgs e)
             }
         }
 
+        /// <summary>
+        /// Finds the LogOutPropterty string to be printed in messages
+        /// </summary>
+        /// <param name="e"> the build event where the LogOutPutProperty string will be added</param>
+        internal void PropertyMapping(LazyFormattedBuildEventArgs e)
+        {
+            string logOutputProperties = "";
+            if (e.BuildEventContext != null)
+            {
+                int nodeId = e.BuildEventContext.NodeId;
+                int projectContextId = e.BuildEventContext.ProjectContextId;
+                propertyOutputMap.TryGetValue((nodeId, projectContextId), out logOutputProperties);
```

Comment:
Under what circumstance would this not return a value? Should that be an internal error instead of silent `string.Empty`?

Reply:
Looking at it now it will always return a value because it is populated in the projectstarted method